eradiate/scenes/measure/_core.py

```diff
@@ -465,36 +502,78 @@ def to_dataset(self, aggregate_spps: bool = False) -> xr.Dataset:
+                    "y": ("y", ys, {"long_name": "film height coordinate"}),
+                    "channel": (
+                        "channel",
+                        ["R", "G", "B"],
+                        {"long_name": "color channel"},
                     ),
-                    "spp": (
-                        [spectral_coord_label, "sensor_id"],
-                        spps,
-                        {"long_name": "sample count"},
+                    spectral_coord_label: (
+                        spectral_coord_label,
+                        spectral_index,
+                        self._to_dataset_spectral_coord_metadata(),
                     ),
-                }
-            ),
-            coords={
-                "x": ("x", xs, {"long_name": "film width coordinate"}),
-                "y": ("y", ys, {"long_name": "film height coordinate"}),
-                spectral_coord_label: (
-                    spectral_coord_label,
-                    spectral_index,
-                    self._to_dataset_spectral_coord_metadata(),
-                ),
-                "sensor_id": (
-                    "sensor_id",
-                    sensor_ids,
-                    {"long_name": "sensor ID"},
+                    "sensor_id": (
+                        "sensor_id",
+                        sensor_ids,
+                        {"long_name": "sensor ID"},
+                    ),
+                },
+            )
+        else:
+            result = xr.Dataset(
+                data_vars=(
+                    {
+                        "raw": (
+                            [spectral_coord_label, "sensor_id", "y", "x", "channel"],
+                            data,
+                            {"long_name": "raw sensor values"},
+                        ),
+                        "spp": (
+                            [spectral_coord_label, "sensor_id"],
+                            spps,
+                            {"long_name": "sample count"},
+                        ),
+                    }
                 ),
-            },
-        )
+                coords={
+                    "x": ("x", xs, {"long_name": "film width coordinate"}),
+                    "y": ("y", ys, {"long_name": "film height coordinate"}),
+                    "channel": (
+                        "channel",
+                        ["intensity"],
+                        {"long_name": "intensity channel"},
+                    ),
+                    spectral_coord_label: (
+                        spectral_coord_label,
+                        spectral_index,
+                        self._to_dataset_spectral_coord_metadata(),
+                    ),
+                    "sensor_id": (
+                        "sensor_id",
+                        sensor_ids,
+                        {"long_name": "sensor ID"},
+                    ),
+                },
+            )
```

Comment:
I'll simplify/streamline that as part of the upcoming post-processing upgrade. In the meantime, we'll keep this conditional.